ci

```diff
@@ -1,88 +0,0 @@
-#!/bin/bash
-
-set -ev # Ref https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps
-set -x
-
-OsLabel="ubuntu"
-if [ "${TRAVIS_OS_NAME:?}" = "osx" ]; then
-    OsLabel="osx-$(sw_vers -productVersion)"
-fi
-
-BuildDir=${3:?}
-DeployEnv=${4:-test}
-Version=$(cat VERSION)
-BuiltPackagePath="${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}".tar.gz
-FinalPackagePath="${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}"-"${OsLabel:?}".tar.gz
-
-# Set default working directory
-cd $BuildDir
-
-case "${1:?}"-"${2:?}" in
-    before_install-dummy)
-        true
-        ;;
-    before_install-*)
-        if [ "${TRAVIS_OS_NAME:?}" = "osx" ]; then
-            export HOMEBREW_CACHE="$HOME/Library/Caches/Homebrew"
-            brew update
-            brew install libsodium
-            # Install really pinned version of OTP Erlang (See #154994576)
-            brew install file://`pwd`/deployment/homebrew/erlang.rb && export PATH="$(brew --prefix erlang)"/bin:"$PATH"
-        elif [ "${TRAVIS_OS_NAME:?}" = "linux" ]; then
-            sudo -H pip install -r deployment/ansible/pip-requirements.txt
-            wget https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz
-            tar zxvf libsodium-1.0.16.tar.gz
-            (cd libsodium-1.0.16; ./configure; make; sudo make install; sudo ldconfig)
-            ansible --version
-        fi
-        erl -version
-        epmd -daemon
-        ;;
-    install-*)
-        true
-        ;;
-    script-dummy)
-        true
-        ;;
-    script-package)
-        make prod-package
-        mv "${BuiltPackagePath:?}" "${FinalPackagePath:?}"
-        # Perform release acceptance testing on the built package.
-        # Configure a simpler PoW than the default one, in order to
-        # speed up mining during the tests.
-        make python-env
-        make python-release-test WORKDIR="$(mktemp -d)" TARBALL="${FinalPackagePath:?}"
-        ;;
-    after_failure-*)
-        LogsDir="${BuildDir}/_build/test/logs"
-        echo "logs/ct.latest.log\n"
-        cat ${LogsDir}/ct.latest.log
-        ls -l ${LogsDir}/latest.sync
-        for f in `ls -1 ${LogsDir}/ct_run*/lib.*.logs/run*/suite.log`; do escript failed_logs.escript $f; done
```

Comment:
failed_logs.escript could be deleted contextually to this PR.

Reply:
deleted in 84cdea4